### DISCUSSION TITLE
How do I get the jk keybinding out of insert mode?

### DISCUSSION THREAD

**Discussion by zetazero1** [NeoVintageous/NeoVintageous#1031] [Q&A]
How do I bind the jk key to exit the insert mode? I plan to press the j and k keys continuously to exit the input mode. How can I achieve this, and where should I configure these configurations? Is it sublime's keymap or is it in my .neovintageousrc file, but I can't find my rc file.

**Comment by gerardroche** [comment]
There is setting to enable this: https://neovintageous.github.io/reference/settings#vintageous-i-escape-jk

There is also a setting for jj and kj too.

Your rc file: see https://neovintageous.github.io/reference/neovintageousrc
